app/controllers/requests/Mappings.scala

```diff
@@ -0,0 +1,24 @@
+package controllers.requests
+
+import business.models.User
+import controllers.requests.auth.CreateUserRequest
+import controllers.requests.user.{GetUserResponse, UpdateUserRequest}
+import reactivemongo.bson.BSONObjectID
+
+/**
+ * Created by liviuignat on 22/03/15.
+ */
+object Mappings {

```

Comment:
I also think this can be dangerous "magical"  code. What if I want to reuse it in more places? I saw others are naming it as *.package.Implicits

Is there a better way to do mappings from one object to another in Scala?